File: dp_labs_and_solutions/original/dp_labs_and_solutions/dp_java_labs_and_solutions/observer/solution/SetupClass.java
// SetupClass.java

public class SetupClass
{
	public static Meeting setup()
	{
		Meeting aMeeting =
			new Meeting(
			4, 3, 2005, 2, 15,
			"Conference Room 3");

		aMeeting.add(new Manager());

		aMeeting.add(new Developer());

		return aMeeting;
	}
}

File: dp_labs_and_solutions/original/dp_labs_and_solutions/dp_matlab_labs_and_solutions/observer/lab/SetupClass.m
% SetupClass.m

classdef SetupClass < handle
	methods (Static)
		function result = setup
			aMeeting = Meeting(4, 3, 2005, 2, 15, 'Conference Room 3');

			aMeeting.addManager(Manager);

			aMeeting.addDeveloper(Developer);

			result = aMeeting;
		end
	end
end
